llarp/router/rc_gossiper.cpp

```diff
@@ -76,17 +76,38 @@ namespace llarp
     DHTImmediateMessage gossip;
     gossip.msgs.emplace_back(new dht::GotRouterMessage(dht::Key_t{}, 0, {rc}, false));
 
-    // send it to everyone
+    constexpr size_t MaxGossipPeers = 20;
```

Comment:
Should be a bit more prominent, at least in the header for this.

Reply:
moved to header